packages/odata-common/src/request-builder/batch/batch-change-set.ts

```diff
@@ -17,11 +18,20 @@ export class BatchChangeSet<
    * @param boundary - Request boundary for separation of sub requests. Defaults to an auto generated value.
    */
   constructor(
-    readonly requests: (
-      | CreateRequestBuilderBase<EntityBase, DeSerializersT>
-      | UpdateRequestBuilderBase<EntityBase, DeSerializersT>
-      | DeleteRequestBuilderBase<EntityBase, DeSerializersT>
-    )[],
+    readonly requests: ChangesetBuilderTypes<DeSerializersT>[],
     readonly boundary: string = `changeset_${uuid()}`
   ) {}
 }
+
+/**
+ *
+ *  Some function imports contain not serializable entities and the execute() method is removed from them.
+ *  Since the execute method is not needed in batch the execute it is removed from all function imports.
+ *
+ * @Internal
```

Comment:
[req] The type is marked internal here but exported in index.ts.

Reply:
Then our pipeline should catch it?